File: src/main/java/fr/ynerant/leveleditor/frame/listeners/OpenMapListener.java
package fr.ynerant.leveleditor.frame.listeners;

import fr.ynerant.leveleditor.api.editor.EditorAPI;
import fr.ynerant.leveleditor.frame.MainFrame;

import java.awt.event.ActionEvent;
import java.awt.event.ActionListener;

public class OpenMapListener implements ActionListener
{
	/* !CodeTemplates.overridecomment.nonjd!
	 * @see java.awt.event.ActionListener#actionPerformed(java.awt.event.ActionEvent)
	 */
	@Override
	public void actionPerformed(ActionEvent event)
	{
		if (EditorAPI.open() != null)
			MainFrame.getInstance().dispose();
	}
}

File: src/main/scala/fr/ynerant/leveleditor/frame/listeners/OpenMapListener.scala
package fr.ynerant.leveleditor.frame.listeners

import java.awt.event.ActionEvent
import java.awt.event.ActionListener

import fr.ynerant.leveleditor.api.editor.EditorAPI
import fr.ynerant.leveleditor.frame.MainFrame


class OpenMapListener extends ActionListener {
	/* !CodeTemplates.overridecomment.nonjd!
		* @see java.awt.event.ActionListener#actionPerformed(java.awt.event.ActionEvent)
		*/ override def actionPerformed(event: ActionEvent): Unit = {
		if (EditorAPI.open != null) MainFrame.getInstance.dispose()
	}
}
